kerlescan/profile_parser.py

```diff
@@ -25,8 +26,22 @@ def _parse_lists_of_strings(names, verb):
         helper method to convert lists of strings to comparable facts
         """
         for list_of_strings in names:
-            for item in system_profile.get(list_of_strings, []):
-                parsed_profile[list_of_strings + "." + item] = verb
+            if verb:
+                for item in system_profile.get(list_of_strings, []):
+                    parsed_profile[list_of_strings + "." + item] = verb
+            else:
```

Comment:
Observation: This new branch could get its own function. Then the `None` wouldn't have to be passed, and no branching/conditional would be necessary.

Reply:
I thought about that, but I think we might be refactoring tags to be multivalue next as well, and I thought maybe one method to handle each of these would be more clear.  I'm happy to split this out into a separate dedicated function if you think that would be clearer to the reader to maintain.

Reply:
Oh, in that case it make sens to keep it as is.